package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "torytechs",
+  "version": "1.0.0",
+  "description": "A community of politically conservative makers and problem solvers",
+  "main": "index.html",
+  "scripts": {
+    "start": "npx http-server",
+    "docker:security": "docker run --rm -e SCAN_URL=\"http://192.168.0.12:8080\" lirantal/is-website-vulnerable:latest",
```

Comment:
I think so...as in, couldnt use the default loopback on the host, since the container thinks its the container loopback...but http-server might actually be listening on any of the other local host IPs

Reply:
Could do with some testing on different machines, in fairness

Reply:
ah right...so looks like there should be a simple more robust way to allow the container to access the host...cause yeah, just tested on another machine and it's 192.168.0.17...lemme work that out and get back to you.

Reply:
...alright, so the short answer is that there's no "simple" solution that works across both linux and non-linux...`host.docker.internal` works for non-linux, but not linux...and setting `--network host` works for linux, but not non-linux...

So the easiest thing is just to have two separate commands, though this is obviously not ideal. Thoughts?

Reply:
I think we can call this an example, specifics are left for further reading :)